samples/maps/demo/geoheatmap/demo.js

```diff
@@ -0,0 +1,176 @@
+(async () => {
+
+    // Load the base world map
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    // Load the leaf area dataset
+    const response = await fetch(
+            'https://cdn.jsdelivr.net/gh/highcharts/highcharts@af1d7a1727/samples/data/geoheatmap-leaf-dataset.json'
+        ),
+        data = await response.json();
+
+    const getGraticule = () => {
+        const data = [];
+
+        // Meridians
+        for (let x = -180; x <= 180; x += 15) {
+            data.push({
+                geometry: {
+                    type: 'LineString',
+                    coordinates: x % 90 === 0 ? [
+                        [x, -90],
+                        [x, 0],
+                        [x, 90]
+                    ] : [
+                        [x, -80],
+                        [x, 80]
+                    ]
+                }
+            });
+        }
+
+        // Latitudes
+        for (let y = -90; y <= 90; y += 10) {
+            const coordinates = [];
+            for (let x = -180; x <= 180; x += 5) {
+                coordinates.push([x, y]);
+            }
+            data.push({
+                geometry: {
+                    type: 'LineString',
+                    coordinates
+                },
+                lineWidth: y === 0 ? 2 : undefined
+            });
+        }
+
+        return data;
+    };
+
+
+    const chart = Highcharts.mapChart('container', {
+
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'GeoHeatmap Leaf Area Index In February 2017',
+            align: 'left',
+            style: {
+                textOutline: '2px white'
+            }
+        },
+
+        subtitle: {
+            text: 'Data source: <a style="color: #666" href="https://neo.gsfc.nasa.gov/view.php?datasetId=MOD15A2_M_LAI">NEO Nasa Earth Observations</a>',
+            align: 'left'
+        },
+
+        mapNavigation: {
+            enabled: true,
+            enableDoubleClickZoomTo: true,
```

Comment:
`enableDoubleClickZoomTo: true,` doesn't look good and has no use for the geoheatmap cells.